File: Sesion 7/Marco teorico/12herencia_virtual_private/12.py
class A:
    def __init__(self, value):
        self.__x = value

    def display(self):
        print("Valor de x:", self.__x)

class B(A):
    def __init__(self, value):
        super().__init__(value)

    def show(self):
        self.display() 

objB = B(10)
objB.show() 

File: Sesion 7/Marco teorico/12herencia_virtual_private/12.java
import java.util.*;

class A {
    private int x;

    public A(int value) {
        x = value;
    }

    public void display() {
        System.out.println("Valor de x: " + x);
    }
}

class B extends A {
    public B(int value) {
        super(value);
    }

    public void show() {
        display(); 
    }
}

public class Main {
    public static void main(String[] args) {
        B objB = new B(10);
        objB.show(); 
    }
}
